src/databricks/labs/ucx/assessment/aws.py

```diff
@@ -67,14 +72,128 @@ def run_command(command):
         return process.returncode, output.decode("utf-8"), error.decode("utf-8")
 
 
+def esc_json_for_cli(json: str) -> str:
+    return json.replace('"', '\\"').replace('\n', '\\n')
+
+
 class AWSResources:
     S3_ACTIONS: typing.ClassVar[set[str]] = {"s3:PutObject", "s3:GetObject", "s3:DeleteObject", "s3:PutObjectAcl"}
     S3_READONLY: typing.ClassVar[str] = "s3:GetObject"
     S3_REGEX: typing.ClassVar[str] = r"arn:aws:s3:::([a-zA-Z0-9+=,.@_-]*)\/\*$"
+    S3_PREFIX: typing.ClassVar[str] = "arn:aws:s3:::"
     UC_MASTER_ROLES_ARN: typing.ClassVar[list[str]] = [
         "arn:aws:iam::414351767826:role/unity-catalog-prod-UCMasterRole-14S5ZJVKOTYTL",
         "arn:aws:iam::707343435239:role/unity-catalog-dev-UCMasterRole-G3MMN8SP21FO",
     ]
+    AWS_ROLE_TRUST_DOC: typing.ClassVar[str] = """
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Principal": {
+                "AWS": "arn:aws:iam::414351767826:role/unity-catalog-prod-UCMasterRole-14S5ZJVKOTYTL"
+            },
+            "Action": "sts:AssumeRole",
+            "Condition": {
+                "StringEquals": {
+                    "sts:ExternalId": "0000"
```

Comment:
All these have to be methods. Don't use strings, but rather dictionaries and serialize to json. Replace variables properly, you may have to prompt users as well

Reply:
Fixed